p2p/module_raintree_test.go

```diff
@@ -178,6 +178,33 @@ func TestRainTreeNetworkCompleteTwentySevenNodes(t *testing.T) {
 	testRainTreeCalls(t, originatorNode, expectedCalls)
 }
 
+//                                                                                                            val_1
```

Comment:
NIT:

12 node test should come before 27 and 18 node test but after 9 node test